cdf_dump/src/main/scala/cognite/spark/cdfdump/SparkHelper.scala

```diff
@@ -0,0 +1,120 @@
+package cognite.spark.cdfdump
+
+import cognite.spark.v1.{CdfSparkException, Constants}
+import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter, Row, SparkSession}
+import cats.syntax.all._
+import org.apache.spark.sql.functions.col
+
+class SparkHelper(
+  writeOptions: Map[String, String],
+  readOptions: Map[String, String],
+  outDir: String,
+  format: String,
+  columns: Option[List[String]],
+  excludeColumns: List[String],
+  filter: Option[String],
+  outPartitions: Option[Int],
+  maxRetries: Option[Int]
+) {
+  val baseUrl = sys.env.getOrElse("COGNITE_BASE_URL", Constants.DefaultBaseUrl)
+  val project = sys.env.get("COGNITE_PROJECT")
+  val auth: Map[String, String] =
+    sys.env.get("COGNITE_API_KEY").map(key => Map("apiKey" -> key))
+      .orElse(
+        (
+          sys.env.get("COGNITE_TOKEN_URI").orElse(sys.env.get("COGNITE_TOKEN_URL")),
+          sys.env.get("COGNITE_CLIENT_ID"),
+          sys.env.get("COGNITE_CLIENT_SECRET")
+        ).mapN((url, id, secret) => Map(
+          "tokenUri" -> url,
+          "clientId" -> id,
+          "clientSecret" -> secret,
+          "scopes" -> sys.env.getOrElse("COGNITE_SCOPES",
```

Comment:
should we also support `audience` here? Some customers have that instead of scopes. not very important though, they are a few.

Reply:
Sure, I'll add COGNITE_AUDIENCE env var. I just didn't know it existed :upside_down_face: